test/app/pages/profile/pages/theme_test.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:bloc_test/bloc_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:talao/app/pages/profile/pages/theme.dart';
+import 'package:talao/theme/cubit/theme_cubit.dart';
+
+import '../../../../helper/pump_app.dart';
```

Comment:
please use full path here. That's just for consistency with the rest of the code

Reply:
ok